crates/sui-core/src/authority_aggregator.rs

```diff
@@ -697,11 +691,13 @@ impl AuthorityAggregator<NetworkAuthorityClient> {
         committee_store: &Arc<CommitteeStore>,
         safe_client_metrics_base: SafeClientMetricsBase,
         auth_agg_metrics: AuthAggMetrics,
-    ) -> anyhow::Result<Self> {
+    ) -> Self {
         // TODO: We should get the committee from the epoch store instead to ensure consistency.
         // Instead of this function use AuthorityEpochStore::epoch_start_configuration() to access this object everywhere
         // besides when we are reading fields for the current epoch
-        let sui_system_state = store.get_sui_system_state_object_unsafe()?;
+        let sui_system_state = store
```

Comment:
did you take a look at whether it would be possible to address the TODO here? If its not easy, I think this expect() is okay anyway (shouldn't be correlated even if it does fail).

Reply:
I have a follow up PR that would this function to be a non-prod function